js/header_dialog.js

```diff
@@ -626,22 +627,26 @@ function HeaderDialog(dialog, onSave) {
 		setParameter('gyro_lowpass2_hz'         ,sysConfig.gyro_lowpass2_hz,0);
 
         if (activeSysConfig.firmwareType == FIRMWARE_TYPE_BETAFLIGHT  && semver.gte(activeSysConfig.firmwareVersion, '4.3.0')) {
-            setParameter('dynNotchCount'           ,sysConfig.dyn_notch_count        , 0);
+            setParameter('dynNotchCount'        ,sysConfig.dyn_notch_count,0);
         } else {
-            setParameter('dynNotchCount'           ,sysConfig.dyn_notch_width_percent, 0);
+            setParameter('dynNotchCount'        ,sysConfig.dyn_notch_width_percent,0);
         }
-        setParameter('dynNotchQ'                   ,sysConfig.dyn_notch_q            , 0);
-        setParameter('dynNotchMinHz'               ,sysConfig.dyn_notch_min_hz       , 0);
-        setParameter('dynNotchMaxHz'               ,sysConfig.dyn_notch_max_hz       , 0);
-
-        setParameter('gyro_rpm_notch_harmonics', sysConfig.gyro_rpm_notch_harmonics  , 0);
-        setParameter('gyro_rpm_notch_q'        , sysConfig.gyro_rpm_notch_q          , 0);
-        setParameter('gyro_rpm_notch_min'      , sysConfig.gyro_rpm_notch_min        , 0);
-        setParameter('rpm_notch_lpf'           , sysConfig.rpm_notch_lpf             , 0);
-
-        setParameter('dterm_rpm_notch_harmonics', sysConfig.dterm_rpm_notch_harmonics, 0);
-        setParameter('dterm_rpm_notch_q'        , sysConfig.dterm_rpm_notch_q        , 0);
-        setParameter('dterm_rpm_notch_min'      , sysConfig.dterm_rpm_notch_min      , 0);
+        setParameter('dynNotchQ'                ,sysConfig.dyn_notch_q,0);
```

Comment:
comma space please

Reply:
it's 'hit and miss' throughout the document, most lines are `,0);` and some are ` 0);`.
I can make them consistent but it will change hundreds of lines :-)